content/posts/irods-development-update-august-2022.md

```diff
@@ -0,0 +1,313 @@
+Title: iRODS Development Update: August 2022
+Date: 2022-08-30 12:00
+Author: Kory Draughn
+Slug: irods-development-update-august-2022
+Status: published
+
+
+Hello everyone! August is on its way out and that means it's time for another development update.
+
+To start things off, TRiRODS was a success. Our interns did a wonderful job on their projects and presentations. If you're interested in hearing about their experience, you can view their talks online at https://irods.org/trirods/.
```

Comment:
please hyperlink the URL...

`[https://irods.org/trirods](https://irods.org/trirods)`